docs/expression_syntax.md

```diff
@@ -163,6 +163,17 @@ Escaped Example
 "/Hello - 'world\//0/\"JsonPointer\""
 ```
 
+### Function
+DataPrepper has some in-built functions that can be used as operand in an expression. For example
+`length(/message) > 20`
+will extract `message` field from the event and compares it's length with 20. The value of `message` field is expected to be of String type. If the field is not present in the event, null is returned and the function is not applied on it.
+Currently the following functions are supported
+ * `length`
+   - takes one argument of string or JsonPointer type
+   - returns the length of the string if the input is of the string type. For example, `length("abcd")` returns 4.
```

Comment:
We are not supporting literals at the moment. The argument must be a jsonPointer